styled/auth.js

```diff
@@ -1,33 +1,71 @@
 import styled from 'styled-components/native';
+import Colors from '../assets/Colors';
 
-export const SignUpContainer = styled.View`
+// TODO @tommypoa: Consider creating re-usable containers taking in props for the 3 Auth screens
+
+// Shared Auth Containers
+
+export const AuthScreenContainer = styled.View`
+  margin: 79px 42px 0 42px;
+  background-color: #fff;
+`;
+
+// StartUpScreen
+// TODO @tommypoa: margin-top should be corrected to sit right above keyboard (as in Login and SignUp screens)
+export const StartUpContainer = styled.View`
   display: flex;
   justify-content: center;
   align-items: center;
   background-color: #fff;
+  margin: 241px 42px 0 42px;
```

Comment:
The set pixel margin of like 241px isn't great on smaller screens like iPhone 8 — switch to percent
![image](https://user-images.githubusercontent.com/21160510/76689727-fdf05380-65f5-11ea-9025-8ed09398f350.png)